js/vm/printjob.js

```diff
@@ -80,18 +83,42 @@ export default class PrintJob {
       price: this.totalPrice,
       printer: this.selectedPrinter,
     };
-    api.post('print', job, {
-      error: (xhr) => {
-        this.status = 'error';
-        if (xhr.status === 507)
-          this.errorText = JSON.parse(xhr.responseText).errors;
-      },
-    }).done(() => {
+
+    // See /api/print in odie-server. Everything's horrible.
```

Comment:
The server only expands on why we're not using GET parameters. My first questions upon reading this hack would be:
* Why have a progress bar in the first place? (avoiding timeouts)
* If you need to send a request body, why not send it... in the request body? (The API won't let us)
* If you need two-way communication, why not use WebSockets? (they're hell to set up right and won't work locally)

Then, the new max printjob size is also not mentioned anywhere. It might never matter, but it *is* a hard technical limitation we're introducing here.
Having the reasons for these decisions here might make posterity understand why we had to stoop to this madness.